### PR TITLE
docs: Remove the banner

### PR DESCRIPTION
**Pull request by jszwedko**
I think since it's been several months and we have "By Datadog" in the
logo now the banner is no longer needed.

I left the partial there so it is easy to bring back with a new announcement if we need to.

Signed-off-by: Jesse Szwedko <jesse@szwedko.me>

### CODE REVIEW

File: docs/layouts/_default/baseof.html [hunk 1]
```diff
@@ -38,7 +38,7 @@
   </head>
   <body class="dark:bg-dark font-sans antialiased flex flex-col min-h-full{{ if $hasBottomNav }} pb-16 lg:pb-0{{ end }}" onload="showBody()" style="visibility: hidden;">
     <div class="flex-1 z-1">
-      {{ partial "banner.html" . }}
+      {#{{ partial "banner.html" . }}#}
```
Comment (lucperkins) [inline docs/layouts/_default/baseof.html:41]: Go template commenting needs to look like this:

```html
{{/* {{ partial "banner.html" . }} */}}
```

**Review by lucperkins — CHANGES_REQUESTED** [review]

File: docs/layouts/_default/baseof.html (hunk 1 above)
Comment (jszwedko) [reply docs/layouts/_default/baseof.html:41]: Ah, you are right. I was relying on my vim plugin but it seems to have picked up that syntax as some other templating language.

Comment (jszwedko) [reply docs/layouts/_default/baseof.html]: Updated in 399006649

File: docs/layouts/_default/baseof.html [hunk 2]
```diff
@@ -38,7 +38,7 @@
   </head>
   <body class="dark:bg-dark font-sans antialiased flex flex-col min-h-full{{ if $hasBottomNav }} pb-16 lg:pb-0{{ end }}" onload="showBody()" style="visibility: hidden;">
     <div class="flex-1 z-1">
-      {{ partial "banner.html" . }}
+      {{ /* {{ partial "banner.html" . }} */}}
```
Comment (lucperkins) [inline docs/layouts/_default/baseof.html:41]: There's an extra space. It needs to be:

```
{{/* {{ partial "banner.html" . }} */}}
```

Comment (jszwedko) [reply docs/layouts/_default/baseof.html]: Ugh, sorry, I should have tested it out. I fixed the spacing and actually ran it locally this time 😄

Comment (spencergilbert) [reply docs/layouts/_default/baseof.html]: Do you love gotmpl yet?

Comment (lucperkins) [reply docs/layouts/_default/baseof.html]: Go templates are actually quite nice for HTML, though a bit counter-intuitive at first. For YAML, on the other hand...

**Review by lucperkins — APPROVED** [review]

File: docs/layouts/_default/baseof.html (hunk 2 above)
Comment (spencergilbert) [reply docs/layouts/_default/baseof.html]: Hah, I'm just glad I get to share some of the misery that's been isolated to the helm charts ;)

**Comment by lucperkins** [comment]
@jszwedko Netlify is having some trouble today. The build hasn't failed, it just won't start. I ran this locally and I'm comfortable merging as-is.

**Pull request merged**